views/dashboard.handlebars

```diff
@@ -25,7 +25,6 @@
                             <label for="sort-order">Sort Ads by: </label>
                         </div>
                         <div id="sortingContainer">
-                            <!-- the stuff above was just me playing around with function and buttons, can be disregarded or deleted -->
                             <!-- below is the actual form for sorting ads based on the filter, currently there's nothing that shows which filter they are set to, and I might have flipped
```

Comment:
This comment can also be removed; that's from Gabby but she fixed the code, so the comment is no longer relevant (I was the reviewer so I remember and I am in part to blame for its still being there 🤭 ).

Reply:
Ok thanks!